pkg/service/credential/storage.go

```diff
@@ -150,6 +148,19 @@ func (cs *Storage) WriteMany(ctx context.Context, writeContexts []WriteContext)
 	return cs.db.WriteMany(ctx, namespaces, keys, values)
 }
 
+func (cs *Storage) IncrementStatusListIndexTx(ctx context.Context, tx storage.Tx) error {
```

Comment:
🤔  All the other methods that are called form the storage files are Exported, they definitely don't need to be, but for matching the rest of the code I'll keep them Uppercase for now (since they are called in the same way as every other method in the storage files).

We can go through and change all to not exported in future PR if needed